elasticsearch/examples/hot-cold/test/goss.yaml

```
@@ -0,0 +1,9 @@
+http:
+  http://localhost:9200/_cluster/health:
```

Calling the service URL is required because goss can execute on any pod including kibana pod
```suggestion
  http://hot-cold-data-hot:9200/_cluster/health:
```